sdv/constraints/base.py

```diff
@@ -135,16 +136,16 @@ def _validate_metadata_columns(cls, metadata, **kwargs):
         else:
             column_names = kwargs.get('column_names')
 
-        missing_columns = set(column_names) - set(metadata._columns)
-
+        missing_columns = set(column_names) - set(metadata._columns) - {None}
         if missing_columns:
+            article = 'An' if cls.__name__ == 'Inequality' else 'A'
             raise ConstraintMetadataError(
-                f'A {cls.__name__} constraint is being applied to invalid column names '
+                f'{article} {cls.__name__} constraint is being applied to invalid column names '
                 f'{missing_columns}. The columns must exist in the table.'
             )
 
-    @classmethod
-    def _validate_metadata_specific_to_constraint(cls, metadata, **kwargs):
+    @staticmethod
```

Comment:
Unrelated. I'm not sure why we were using classmethod? It seems to me that all child classes can be set to staticmethods, so I did so.

Reply:
should we change it to match in the base as well?